modules/telco-core-security.adoc

```diff
@@ -1,31 +1,66 @@
+* **Seccomp**: All pods should run with the `RuntimeDefault` (or stronger) seccomp profile.
+* **Rootless DPDK pods**: Many user-plane networking (DPDK) CNFs require pods to run with root privileges.
+With this feature, a conformant DPDK pod can run without requiring root privileges.
 Rootless DPDK pods create a tap device in a rootless pod that injects traffic from a DPDK application to the kernel.
-* **Storage**: The storage network should be isolated and non-routable to other cluster networks. See the "Storage" section for additional details.
+* **Storage**: The storage network should be isolated and non-routable to other cluster networks.
+See the "Storage" section for additional details.
+
+Refer to link:https://access.redhat.com/articles/7090422[Custom nftable firewall rules in OpenShift] for a supported method for implementing custom nftables firewall rules in OpenShift cluster nodes.
+This article is intended for cluster administrators who are responsible for managing network security policies in OpenShift environments.
+
+It is crucial to carefully consider the operational implications before deploying this method, including:
+
+* **Early application**: The rules are applied at boot time, before the network is fully operational.
+Ensure the rules don't inadvertently block essential services required during the boot process.
+* **Risk of misconfiguration**: Errors in your custom rules can lead to unintended consequences, potentially leading to performance impact or blocking legitimate traffic or isolating nodes.
+Thoroughly test your rules in a non-production environment before deploying them to your main cluster.
+* **External endpoints**: OpenShift requires access to external endpoints to function.
+For more information about the firewall allowlist, see "Configuring your firewall for {product-title}".
+
+Ensure that cluster nodes are permitted access to those endpoints.
+* **Node reboot**: Unless node disruption policies are configured, applying the `MachineConfig` CR with the required firewall settings causes a node reboot.
+Be aware of this impact and schedule a maintenance window accordingly.
+[NOTE]
+====
+Node disruption policies are available in {product-title} 4.17 and later.
+====
+
+* **Network flow matrix**: For more information about managing ingress traffic, see "{product-title} network flow matrix".
+You can restrict ingress traffic to essential flows to improve network security.
+The matrix provides insights into base cluster services but excludes traffic generated by Day-2 Operators.
+
+* **Cluster version updates and upgrades**: Exercise caution when updating or upgrading OpenShift clusters.
+Recent changes to the platform's firewall requirements might require adjustments to network port permissions.
+While the documentation provides guidelines, note that these requirements can evolve over time.
```

Comment:
Per ISG: [Use _while_ only to refer to a period of time. To avoid potential ambiguity, do not use the term as a synonym for although or though](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#w).

```suggestion
Although the documentation provides guidelines, note that these requirements can evolve over time.
```